azure-functions/acbs-function/mappings/facility/facility-covenant-amend.test.ts

```diff
@@ -0,0 +1,57 @@
+import { facilityCovenantAmend } from './facility-covenant-amend';
```

Comment:
Really good test coverage, please keep it to `JS`.

Reply:
any reason why we'd want to keep this js?